docs/extensions/storage/_index.md

```diff
@@ -8,39 +8,144 @@ geekdocFilePath: _index.md
+|`home-alice:56f7ceca-e7f8-4530-9a7a-fe4b7ec8089a:` | in the *storage space* `home-alice`, start at the resource identified by the `logical uuid` `56f7ceca-e7f8-4530-9a7a-fe4b7ec8089a`, no relativ path to follow |
+|`ee1687e5-ac7f-426d-a6c0-03fed91d5f62:56f7ceca-e7f8-4530-9a7a-fe4b7ec8089a:` | in the *storage space* `ee1687e5-ac7f-426d-a6c0-03fed91d5f62`, start at the resource identified by the `logical uuid` `56f7ceca-e7f8-4530-9a7a-fe4b7ec8089a`, no relativ path to follow (id based reference) |
+
+All those examples reference the same *resource* (if the `logical uuid` of `projects` is 
+`c3cf23bb-8f47-4719-a150-1d25a1f6fb56` and the `logical uuid` of `foo` is `56f7ceca-e7f8-4530-9a7a-fe4b7ec8089a`).
+To move *storage spaces* between *storage providers*, migrate files or restore backups, the `logical uuid` of a file must be writable.
+{{< /hint >}}
+
+
+### Namespaces
+A *namespace* is a set of paths with a common prefix. There is a global namespace when making requests against the CS3 *gateway* and a local namespace for every *storage provider*. Currently, oCIS mounts these *storage providers* out of the box:
+
+| mountpoint | served namespace |
+|------------|------------------|
+| `/home`    | currently logged in users home |
+| `/users`   | all users, used to access collaborative shares |
+| `/public`  | publicly shared files, used to access public links |
+
+
+{{< hint warning >}}
+We plan to serve all shares that the current user has access to under `/shares`.
+{{< /hint >}}
+
+### Storage Drivers
+
+A *storage driver* implements access to a *storage system*:
+
+It maps the *path* and *id* based CS3 *references* to an appropriate *storage system* specific reference, e.g.:
+- eos file ids
+- posix inodes or paths
+- deconstructed filesystem nodes
+
+### Storage Provider
+
+A *storage provider* manages *resources* identified by a *reference*
+by accessing a *storage system* with a *storage driver*.
+
+{{< svg src="extensions/storage/static/storageprovider.drawio.svg" >}}
+
+{{< hint warning >}}
+**Proposed Change**
+A *storage provider* manages multiple *storage spaces*
+by accessing a *storage system* with a *storage driver*.
+
+{{< /hint >}}
+{{< svg src="extensions/storage/static/storageprovider-spaces.drawio.svg" >}}
+{{< hint warning >}}
+By making *storage providers* aware of *storage spaces* we can get rid of the current `enablehome` flag / hack in reva. Furthermore, provisioning a new *storage space* becomes a generic operation, regardless of the need of provisioning a new user home or a new project space.
+{{< /hint >}}
+
+### Storage Registries
+
+A *storage registry* manages the global *namespace*:
+It is used by the *gateway*
+to look up `address` and `port` of the *storage provider*
+that should handle a *reference*.
+
+{{< svg src="extensions/storage/static/storageregistry.drawio.svg" >}}
+
+{{< hint warning >}}
+**Proposed Change**
```

Comment:
Can't build the docs locally, but I guess it won't render the correct linebreak here since you're going for sort-of-a-heading. You may want to go for a h4/h5 (`#### Proposed Changes`) or 

```markdown
**Proposed Change:**


...xyu
```